webapp/components/ImageUploader/ImageUploader.vue

```diff
@@ -73,7 +74,20 @@ export default {
     onDropzoneError(file, message) {
       this.$toast.error(file.status, message)
     },
+
+    onUnSupportedFormat(status, message) {
+      this.$toast.error(status, message)
+    },
     initCropper(file) {
+      const supportedFormats = ['image/jpg', 'image/jpeg', 'image/png', 'image/gif']
+
+      if (supportedFormats.indexOf(file.type) < 0) {
+        this.onUnSupportedFormat(
+          'error',
+          'Please upload an image of file format : jpg , jpeg , png or gif',
```

Comment:
Please lets use locales for translations here @dasari810 🙏🏼
That the users have their languages.

Would be nice if you can write the possible file formats as capitals like in your other PR #3925 . 🤗